com.github.eneshecan.WhatsApp-For-Linux.yml

```diff
@@ -0,0 +1,146 @@
+app-id: com.github.eneshecan.WhatsApp-For-Linux
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: whatsapp-for-linux
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+# - --device=dri # OpenGL rendering
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=com.github.eneshecan.whatsapp-for-linux
```

Comment:
Given that in the code it's `com.github.whatsapp-for-linux` this is likely wrong. (the code is wrong too because the app id doesn't match the requirement, but unless you patch it, it is what it is)

Other notes: by default Flatpak grant "own name" based on the package ID, as it's case sensitive.

Reply:
Okay, I see now. I will fix it in the application code and here then. What is the best app-id then?
* com.github.eneshecan.whatsapp-for-linux
* com.github.eneshecan.Whatsapp-For-Linux
* com.github.eneshecan.WhatsappForLinux